pipelines/migrate_bucket_regions/migrate_bucket_data_via_local.wdl

```diff
@@ -0,0 +1,138 @@
+version 1.0
+
+workflow migrate_data_via_local {
+    input {
+        String source_bucket_path
+        String destination_bucket_path
+
+        File   source_bucket_object_inventory
```

Comment:
Ah, I see. Yes, you wrote that the BQ table will be added as an input to the WDL. Amazing!